activiti-engine/src/test/java/org/activiti/engine/test/impl/util/json/JSONObjectTest.java

```diff
@@ -0,0 +1,179 @@
+package org.activiti.engine.test.impl.util.json;
+
+import org.activiti.engine.impl.util.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class JSONObjectTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: JSONObject.java */
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 1024 branch to line 1024
+   *  - conditional line 1024 branch to line 1025
+   */
+
+  @Test
+  public void quoteInputNotNullOutputNotNull() {
+
```

Comment:
:+1: It would have been nicer had it put something other than an empty string in, but saying that, this could be an edge case, which is nice.